brainiak/fcma/preprocessing.py

```diff
@@ -292,8 +374,20 @@ def prepare_searchlight_mvpa_data(images, conditions, data_type=np.float32):
 
     for sid, f in enumerate(images):
         data = f.get_data().astype(data_type)
+        [d1, d2, d3, d4] = data.shape
+        if random == RandomType.REPRODUCIBLE:
+            data = _randomize_single_subject(data.reshape((d1 * d2 * d3, d4)),
```

Comment:
I didn't do this because I would like to make the input argument `data` of `_randomize_single_subject` an nVoxels x nTRs 2D array for clarity purpose.